org-code-codebuilderapp/app/build.gradle

```diff
@@ -0,0 +1,45 @@
+/*
+ * Usage:
+ * gradle build -> builds the project
+ * gradle bootRun -> sets up a local server -> can be accessed at localhost:8080
+ *
+ * To get your changes into the docker container, do the following
+ * gradle build
+ * cp app/build/libs/app-0.0.1-SNAPSHOT.jar dockerContainer
+ * cd dockerContainer
+ * docker build .
+ *
+ * To publish that docker container, do the following
+ * docker tag <id> jmkulwik/test-fargate-websocket:<tag>
+ * docker push jmkulwik/test-fargate-websocket:<tag>
+ */
+
+plugins {
+  id 'org.springframework.boot' version '2.4.2'
+  id 'io.spring.dependency-management' version '1.0.11.RELEASE'
+  id 'java'
+}
+
+group = 'org.code'
+version = '0.0.1-SNAPSHOT'
+sourceCompatibility = '1.8'
+
+repositories {
+  mavenCentral()
+}
+
+dependencies {
+  implementation 'org.springframework.boot:spring-boot-starter-websocket'
+  implementation 'org.webjars:webjars-locator-core'
+  implementation 'org.webjars:sockjs-client:1.0.2'
```

Comment:
Is there a way to have the demo UI be more separate from the rest of the app?  I'm thinking that we'll ultimately want to ship the service without the UI (right?) and it would be good to keep the js dependencies out of the main list here.  Would it be too much work to build the js project completely separately from the java code?

Reply:
Hmm... I'm planning to remove the demo UI as soon as we integrate with code studio (i.e. over the next sprint or two). Probably wouldn't be worth it to invest in separating it for such a short period of time.

Reply:
Although now I'm thinking of how nice it was to have a minimal client application that I could use for testing. May be nice to keep that long-term.